src/index.html

```diff
@@ -585,6 +47,9 @@
                     <span class="ver">{{version}}</span>
                 </a>
             </nav>
+            <button class="dark-mode-trigger" onclick="localStorage.setItem('mode', (localStorage.getItem('mode') || 'dark') === 'dark' ? 'light' : 'dark'); localStorage.getItem('mode') === 'dark' ? document.querySelector('body').classList.add('dark') : document.querySelector('body').classList.remove('dark')">
+                <img width="16px" height="16px" src='data:image/svg+xml;utf8,<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="adjust" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-adjust fa-w-16 fa-2x"><path fill="currentColor" d="M8 256c0 136.966 111.033 248 248 248s248-111.034 248-248S392.966 8 256 8 8 119.033 8 256zm248 184V72c101.705 0 184 82.311 184 184 0 101.705-82.311 184-184 184z" class=""></path></svg>' alt="dark mode" />
```

Comment:
Should we use the svg directly here? Is there something else going on that won't let use use the svg?

Reply:
I tried to do it as it was above with the github logo 😅😂.

Reply:
Oopsie! 😄 I guess it's fine as it is then -- sorry forgot the previous stuff.